demo/data.js

```diff
@@ -406,9 +406,74 @@ export const demoData = {
         R174: "Zimbabwe",
         S1: "Value",
         S2: "=ROUND(RANDARRAY(ROWS(R2:R174) -1)*1000)",
+        V1: "Movie Genre",
+        V2: "Action",
+        V3: "Action",
+        V4: "Action",
+        V5: "Action",
+        V6: "Action",
+        V7: "Animation",
+        V8: "Animation",
+        V9: "Animation",
+        V10: "Animation",
+        V11: "Animation",
+        V12: "Drama",
+        V13: "Drama",
+        V14: "Drama",
+        V15: "Drama",
+        V16: "Drama",
+        V17: "Action",
+        V18: "Animation",
+        V19: "Drama",
+        V20: "Action",
+        V21: "Animation",
```

Comment:
V22: "Misc",
V23: "Misc"

Add a category with a single level to display the full sunburst visuals